src/msw/bitmap.cpp

```diff
@@ -1290,6 +1290,32 @@ void wxBitmap::SetMask(wxMask *mask)
     GetBitmapData()->SetMask(mask);
 }
 
+void wxBitmap::SetHBITMAP(WXHBITMAP bmp, int width, int height, int depth)
+{
+#if wxDEBUG_LEVEL
```

Comment:
Is `::GetObject()` really fast/free? If not, I'd make this check `wxDEBUG_LEVEL >= 2` i.e. avoid always paying its price in the default build with `wxDEBUG_LEVEL == 1` as we don't want to pessimize bitmap code, this is one of the places where the speed does matter.